packages/truffle-db/src/workspace/pouch.ts

```diff
@@ -69,10 +78,50 @@ export class Workspace {
     }
   }
 
+  private async fetchAll(res : IWorkspaceQueryResource): Promise<IWorkspaceQueryResourceCollection> {
+    await this.ready;
+
+    try {
+      const query = { selector: {} }
+      const { docs } : any = await this.dbApi[res].find(query);
+
+      return docs.map(doc => ({...doc, id: doc['_id']}));
```

Comment:
I might be misunderstanding this line, but there is a difference between `id`(we generate this for each resource)  and `_id`(pouchDB generates this). Not sure if this is the one you want to return.

Reply:
This makes sure the returned object has an `id` property by mapping pouchDB's `_id` to `id` to conform with GraphQL schema.

Reply:
Ah, never mind. Our internal `id` is actually set to `_id` when each resource is added to workspace. They are actually the same value.